### PR TITLE
Create a resource group and azure monitor workspace to store k6tester metrics

### PR DESCRIPTION
**Pull request by monteiro-renato**
## Related Issue(s)
- #987 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new resource group and monitor workspace in Azure for enhanced monitoring capabilities.

### CODE REVIEW

File: infrastructure/adminservices-test/altinn-monitor-test-rg/lasttest.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+resource "azurerm_resource_group" "lasttest_rg" {
+  name     = "lasttest-rg"
+  location = "norwayeast"
+}
+
+resource "azurerm_monitor_workspace" "lasttest_amw" {
+  name                = "lasttest-amw"
+  resource_group_name = azurerm_resource_group.lasttest_rg.name
+  location            = azurerm_resource_group.lasttest_rg.location
+}
```
[the remark this answers is not in the document]
Comment (monteiro-renato) [reply infrastructure/adminservices-test/altinn-monitor-test-rg/lasttest.tf:10]: Not relevant

File: infrastructure/adminservices-test/altinn-monitor-test-rg/lasttest.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+resource "azurerm_resource_group" "lasttest_rg" {
+  name     = "lasttest-rg"
+  location = "norwayeast"
+}
```
[the remark this answers is not in the document]
Comment (monteiro-renato) [reply infrastructure/adminservices-test/altinn-monitor-test-rg/lasttest.tf:4]: It's a test for the time being.
We want to test it works and that it fulfills the requestor's needs.

File: infrastructure/adminservices-test/altinn-monitor-test-rg/lasttest.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (monteiro-renato) [reply infrastructure/adminservices-test/altinn-monitor-test-rg/lasttest.tf:10]: I will create the SP in another PR.

**Review by khanrn — APPROVED** [review]
🥇

**Pull request merged**